docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -72,14 +92,62 @@ the sample results by providing a valid interval for your data. The generated
 preview contains the number of potentially created alerts during the relative 
 time range you defined.
 
+As the last step in the rule creation process, 
+<<defining-actions, define the actions>> that occur when the conditions
+are met.
+
+
+[[creating-anomaly-jobs-health-rules]]
+=== {anomaly-jobs-cap} health
+
+Select the job or group that
+the rule applies to. If you assign more jobs to the group, they are
+included the next time the rule conditions are checked.
+
+You can also use a special character (`*`) to apply the rule to all your jobs. 
+Jobs created after the rule are automatically included. If you have many jobs, 
+you might receive numerous notifications. To avoid this situation, you can 
```

Comment:
We group alert instances by the health check type, so this sentence might be misleading.